mesonbuild/compilers/compilers.py

```diff
@@ -342,6 +344,8 @@ def get_base_compile_args(options: 'KeyedOptionDictType', compiler: 'Compiler',
 def get_base_link_args(options: 'KeyedOptionDictType', linker: 'Compiler',
                        is_shared_module: bool, build_dir: str) -> T.List[str]:
     args: T.List[str] = []
+    if mesonlib.is_os2():
+        args += ['-Zomf']
```

Comment:
These should go in the `Compiler.get_always_args` and `Linker.get_always_args` methods.

Reply:
Ok, I'll try.

Reply:
I'm trying to move those codes to GnuCompiler. However, I don't know how to access to the global options. How can I do that?